pkg/kudoctl/cmd/install/install.go

```diff
@@ -73,7 +79,13 @@ func installOperator(operatorArgument string, options *Options, fs afero.Fs, set
 
 	clog.V(3).Printf("getting operator package")
 
-	resolver := pkgresolver.New(repository)
+	var resolver pkgresolver.Resolver
```

Comment:
This `if-else` will probably be somewhat controversial: [PackageResolver](https://github.com/kudobuilder/kudo/blob/main/pkg/kudoctl/packages/resolver/resolver.go#L19) returned by the `pkgresolver.New` method is already on overarching resolver combining the others (repo/folder/URL) under one `Resolve(...)` method call. So, `InClusterResolver` could, at least in theory, become one of them. However, given that when the `--in-cluster` option is used, **no other** resolver should be even considered, I felt that it would make more sense to keep it separately.